Quintet-iOS/View/CalendarView.swift

```diff
@@ -0,0 +1,157 @@
+                }) {
+                    viewModel.yearMonthButtonTextRecordVer
+                        .padding(.horizontal)
+                        .foregroundColor(Color.black)
+                    
+                }.sheet(isPresented: $isShowPopup) {
+                    CalendarYearMonthPickerPopup(viewModel: viewModel, isShowPopup: $isShowPopup)
+                        .frame(width: 300, height: 400)
+                        .background(BackgroundClearView())
+                        .ignoresSafeArea()
+                }
+                
+                Image(systemName: isShowPopup ? "chevron.up" : "chevron.down")
+                    .foregroundColor(.black)
+                    .font(.system(size: 15))
+            }
+            .padding(.bottom, 17)
+            
+            HStack(spacing: 2) {
+                
+                ForEach(days, id: \.self) { day in
+                    
+                    Text(day)
+                        .fontWeight(.medium)
+                        .padding(.horizontal)
+                        .foregroundColor(Color("DarkGray"))
+                    
+                }
+            }
+            
+            Divider()
+                .padding(.horizontal)
+            
+            let columns = Array(repeating: GridItem(.fixed(41)), count: 7)
+            LazyVGrid(columns: columns, spacing: 9) {
+                ForEach(extractDate()) { value in
+                    if value.day != -1 {
+                        CardView(value: value)
+                            .background (
+                                Circle()
+                                    .frame(width: 40, height: 40)
+                                    .foregroundColor(Color("DarkQ"))
+                                    .opacity(isSameDay(date1: value.date, date2: currentDate) ? 1 : 0)
+                            )
+                            .onTapGesture {
+                                currentDate = value.date
+                            }
+                    } else {
+                        Color.clear
+                    }
+                }
+            }
+            .fontWeight(.light)
+        }
+    
+        if let task = coreDataViewModel.getRecordMetaData(selectedYear: viewModel.selectedYear, selectedMonth: viewModel.selectedMonth).first (where: { task in
+                return isSameDay(date1: task.date, date2: currentDate)
+            }) {
+                VStack {
+                    Text("오늘의 5요소")
+                        .font(.system(size: 23))
+                        .fontWeight(.semibold)
+                        .padding(.trailing, 190)
+                        .padding(.bottom, 20)
+                        .padding(.top, 10)
+                        .foregroundColor(Color.black)
+                    
+                    ForEach(task.records) { record in
+                        recordCard(icon: record.icon, title: record.title, subtitle: record.subtitle)
```

Comment:
```suggestion
                    ForEach(task.records) { record in
                        recordCard(
                            icon: record.icon,
                            title: record.title,
                            subtitle: record.subtitle
                        )
```

와 같이 작성하면 가독성이 더 좋아질 것 같습니다.! 또는 아래와 같은 방식은 어떨까요?

```suggestion
                    ForEach(task.records) {
                        recordCard(
                            icon: $0.icon,
                            title: $0.title,
                            subtitle: $0.subtitle
                        )
```